lib/membrane_webm/parser/element.ex

```diff
@@ -0,0 +1,260 @@
+defmodule Membrane.WebM.Parser.Element do
+  @moduledoc """
+  An EBML element consists of
+  - element_id - a VINT
+  - element_data_size - a VINT
+  - element_data - which is another element
+    - signed integer element
+    - unsigned integer element
+    - float element
+    - string element
+    - UTF-8 element
+    - date element
+    - binary element
+      contents should not be interpreted by parser
+    - master element
+      A Master Element declare a length in octets from zero to VINTMAX or be of unknown
+      length. See Section 6 for rules that apply to elements of unknown length.
+      The Master Element contains zero or more other elements. EBML Elements contained within a
+      Master Element have the EBMLParentPath of their Element Path equal to the
+      EBMLFullPath of the Master Element Element Path (see Section 11.1.6.2). Element Data stored
+      within Master Elements only consist of EBML Elements and contain any
+      data that is not part of an EBML Element. The EBML Schema identifies what Element IDs are
+      valid within the Master Elements for that version of the EBML Document Type. Any data
+      contained within a Master Element that is not part of a Child Element be ignored.
+
+  TODO element_id is the only type of VINT for which it is illegal to take up more space than is necessary i.e.
+    1 0000001 is legal
+    0 1 00000000000001 is illegal because a shorter encoding of VINT_DATA is available
+    (it fits in 1 byte but 2 are used)
+
+  element_data_size can be unknown!
+    https://www.rfc-editor.org/rfc/rfc8794.pdf section 6.2
+    craaayyyzy
+
+
+  """
+
+  alias Membrane.Time
+  alias Membrane.WebM.Parser.Vint
+  alias Membrane.WebM.Schema
+
+  def parse_list(bytes, acc) do
```

Comment:
How all that is different from the code in Parser itself? Seems to be duplicated.